### ISSUE TITLE
All exception raised need to be logged in a trace file

### ISSUE THREAD

**Issue by jLeta** [ETH-WORKS-DASHBOARD/HN_HR_1.0#2]
We need to clean up the code so that every potential call that may raise exception need to be in a try catch block. and once we catch exception we will need to log it to a trace file. :v:

**Comment by jLeta** [comment]
I think there are still some manager class that are not handling exception

**Comment by Jdan116** [comment]
Yes there is a manager that are not handling exception. I will check this out.

**Comment by jLeta** [comment]
:+1:
